#goal definition
int32 start_angle
int32 end_angle
int32 distance_of_scan
---
#result definition
int32 angle_found
int32 distance_to_sample
---
#feedback
int32 current_angle